src/main/java/org/owasp/esapi/reference/DefaultSecurityConfiguration.java

```diff
@@ -627,6 +627,64 @@ public File getResourceFile(String filename) {
 		return null;
 	}
 	
+    /**
+     * Used to load antisamy-esapi.xml from a variety of different classpath locations.
```

Comment:
If the intent is to make this available in the `SecurityConfiguration` interface, it would be better to just use
{@inheritDoc}
for the Javadoc here so that they don't get out of sync in the two places. (Better to have a single source of record.) You also can use that and then add some additional documentation if you feel that necessary.

Reply:
accepted, but waiting on the other comments to update the PR

Reply:
This is a moot point now, since no new interfaces were added to SecurityConfiguration.